### PR TITLE
tests: add sseramemes first test

### PR DESCRIPTION
**Pull request by arantespp**
# Summary

Configure and add the first test for sseramemes bot. The test check which mime types can apply the logo.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: apps/sseramemes/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.png
```
Comment (sibelius) [inline apps/sseramemes/.gitignore:1]: does this .gitignore works?

**Review by sibelius — APPROVED** [review]
cool

File: apps/sseramemes/.gitignore (hunk 1 above)
Comment (arantespp) [reply apps/sseramemes/.gitignore:1]: Yes, it works.

image

File: apps/sseramemes/jest.config.ts [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/tmp/jest_rt",
+
+  // Automatically clear mock calls, instances, contexts and results before every test
+  clearMocks: true,
```
[the remark this answers is not in the document]
Comment (arantespp) [reply apps/sseramemes/jest.config.ts:17]: I run `jest --init` to generate this file automatically. Do you prefer I remove the comments?

**Pull request merged**